README.rst

```diff
@@ -183,18 +183,75 @@ For information on all the parameters and their usage, run
   $ cortex circuit run --help
 
 
-The results of the measurements in the circuit are returned in JSON format:
+The results of all the measurements in the circuit can be returned in human-readable or machine-readable format:
 
-.. code-block:: json
+``--output frequencies`` provides a human-readable output of the measurement frequencies, this is the default format:
 
-  {"measurement_0":
-    [
-      [1, 0, 1, 1],
-      [1, 0, 0, 1],
-      [1, 0, 1, 1]
-    ]
-  }
+.. code-block:: bash
+
+  $ cortex circuit run --output frequencies --shots 5000 --iqm-json ./tests/resources/valid_circuit_2.json
+  Circuit "Serialized from Cirq" results using calibration set 41 over 5000 shots:
```

Comment:
Let's change circuit name to something else

Reply:
Updated

Reply:
I think "Serialized from Cirq" is meant, which I would also suggest to change.

Reply:
And that one is updated as well